src/main/java/com/machowina/repository/ParkingZoneRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.machowina.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import com.machowina.model.ParkingZone;
+
+@Repository
+public interface ParkingZoneRepository extends JpaRepository<ParkingZone, Long> {
+
+	ParkingZone findFirstBy();
```

Comment:
findFirstBy co ? :)

Reply:
Zwróć pierszą znalezioną strefę:) To było uzywane w metodzie z której zrezygnowałam, zapomniałam usunąć